MDANSE_GUI/Src/MDANSE_GUI/Tabs/Models/PlotDataModel.py

```diff
@@ -176,7 +199,7 @@ def add_file(self, filename: str):
         try:
             new_datafile = MDADataStructure(filename)
         except Exception as e:
-            LOG.error(f"Invalid: {e!s}")
+            LOG.error(f"Invalid: {e}")
```

Comment:
Strictly this should be
```suggestion
            LOG.error(f"Invalid: %s", e)
```